src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -55,28 +55,54 @@ public boolean extractQuestionDetails(
         List<String> rubricSubQuestions = new ArrayList<String>();
         List<List<String>> rubricDescriptions = new ArrayList<List<String>>();
```

Comment:
These don't need to be initialized first. You can just do it like this:
```java
List<String> rubricChoices = getRubricChoices(...);
```
when they're used later on.